### PR TITLE
fix: fixed react flow link

### LINKED ISSUE #4266: 🐛 Bug Report: [Quick start guide] Getting started page : React flow button UI fixing.
### 📜 Description

When we visit the getting started page , the React Flow anchor tag link is not clearly visible finds makes user experience unsatisfactory.
Screenshot 2023-10-01 091452

Color overlaps with the element behind and two links are placed very close to each other which does not look good.

### 👟 Reproduction steps

1. Go to the Getting started page.
2. Look down for the Element having a title 'Channel' and description 'Build desired order of channels' .
3. In the same element there is a pink colored link which looks great .
4. Below that you will see the React Flow link which is not clearly visible.
Screenshot 2023-10-01 091452

### 👍 Expected behavior

The React Flow link which redirects to [https://reactflow.dev/](url) should have seperate position and should be highlighted either with a lighter shade or a different color. The link should appear distant from background element.

### 👎 Actual Behavior with Screenshots

Screenshot 2023-10-01 091452

### Novu version

Novu Saas

### npm version

_No response_

### node version

_No response_

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by govardhanshah456**
### What change does this PR introduce?

Made the React Flow Link More foundable.

### Why was this change needed?

closes #4266 

### Other information (Screenshots)
image

### CODE REVIEW

**Comment by scopsy** [comment]
@govardhanshah456 could you please keep the original color of the link and just move the position to be centered as you did but with 30px top margin?

**Comment by govardhanshah456** [comment]
I was going to keep original color only but then it is barely visible in dark mode so I changed it to deeppink:(image for refn)
image
when keeping hover one can notice but only when hover event occurs(image refn)
image

So should i revert it back to original color considering above conditions?

**Comment by govardhanshah456** [comment]
> I see, could you please just move it to the center with the old color? And just moving it down by 30px?

You can refer the latest commit, I have made that changes on it only

**Comment by scopsy** [comment]
@govardhanshah456 mind sharing a photo of how it looks after the changes?

**Comment by govardhanshah456** [comment]
> @govardhanshah456 mind sharing a photo of how it looks after the changes?

image

**Comment by govardhanshah456** [comment]
@scopsy any other changes required?

File: apps/web/src/components/quick-start/common/Workflow.styles.tsx [hunk 1]
```diff
@@ -12,11 +12,28 @@ export const WorkflowWrapper = styled.div<{ height: string }>`
...
   .react-flow__attribution {
     background: transparent;
     opacity: 0.5;
   }
 
+  .react-flow__attribution a {
+    position: relative;
+    text-decoration: underline;
+    font-size: initial;
+    top: 11px;
```
Comment (scopsy) [inline apps/web/src/components/quick-start/common/Workflow.styles.tsx:28]:
```suggestion
    top: 25px;
```

File: apps/web/src/components/quick-start/common/Workflow.styles.tsx [hunk 2]
```diff
@@ -12,11 +12,28 @@ export const WorkflowWrapper = styled.div<{ height: string }>`
...
+    right: 173px;
+  }
+
+  .react-flow__attribution a:hover {
+    color: lightpink !important;
+    text-decoration: underline;
```
Comment (scopsy) [inline apps/web/src/components/quick-start/common/Workflow.styles.tsx:34]:
```suggestion
```

File: apps/web/src/components/quick-start/common/Workflow.styles.tsx [hunk 3]
```diff
@@ -12,11 +12,28 @@ export const WorkflowWrapper = styled.div<{ height: string }>`
...
+  }
+
   .react-flow__attribution {
     background: transparent;
     opacity: 0.5;
   }
 
+  .react-flow__attribution a {
+    position: relative;
+    text-decoration: underline;
```
Comment (scopsy) [inline apps/web/src/components/quick-start/common/Workflow.styles.tsx:26]:
```suggestion
```

**Review by scopsy — CHANGES_REQUESTED** [review]
Just a small adjustment here

**Comment by scopsy** [comment]
@govardhanshah456 please comment on the original issue so I can assign you

**Comment by govardhanshah456** [comment]
> @govardhanshah456 please comment on the original issue so I can assign you

yes, did that, and commited the suggestions provided

**Review by scopsy — APPROVED** [review]

**Comment by govardhanshah456** [comment]
@scopsy Hey, can you assign this to me? i have commented on the issue as well for the same.

Thanks

**Pull request merged**